.buildkite/test-pipeline.yaml

```diff
@@ -205,6 +205,24 @@ steps:
   - VLLM_ALLOW_INSECURE_SERIALIZATION=1 RAY_DEDUP_LOGS=0 python3 rlhf_colocate.py
   - popd
 
+- label: Distributed Tests (8 GPUs) # ?min
```

Comment:
do we have the capacity to run this?

Reply:
Seems not https://github.com/vllm-project/ci-infra/blob/a64d73f396ba8629e8fb9c5e5b933a5f87d1edc2/buildkite/pipeline_generator/utils.py#L32
Possible to add 8-card capacity? @hl475

Reply:
if we want to use 8-card capacity, maybe try with H100? example PR https://github.com/vllm-project/vllm/pull/27113/files